src/test/pages/workspaces/List.test.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { configure, mount } from 'enzyme'
+import { WorkspaceList } from 'src/main/pages/workspaces/List'
+import Adapter from 'enzyme-adapter-react-16'
+import * as Main from 'src/main/pages/Main'
+import { mockRawls } from 'src/test/mock/mockRawls'
+
+
+configure({ adapter: new Adapter() })
+
+Main.initNavPaths()
+
+describe('WorkspaceList', () => {
+  it('should render a TopBar and DataGrid', () => {
+    const wrapper = mount(React.createElement(WorkspaceList, { Rawls: mockRawls }))
+    expect(wrapper.find('TopBar').length).toEqual(1)
+    expect(wrapper.find('DataGrid').length).toEqual(1)
```

Comment:
These assertions seem tedious to me since they essentially mimic the form of the code in the `render` function. I'd hate to have to, in some sense, re-type my code here after adding more there.

Reply:
I just meant this as an example. This test has little value, but the other one seems decent (aside from the hacky way to find things).